src/Components/Admin/common/itemCard.js

```diff
@@ -17,6 +17,20 @@ const useStyles = makeStyles(() => ({
         maxWidth: 400,
         marginTop: 16,
     },
+    metadata: {
+        display: "flex",
+        justifyContent: "space-between",
+        marginLeft: "5.7%",
+        marginRight: "5.7%",
```

Comment:
@ahmedhamodi is this best practice, doing decimals with percentages?

Reply:
I think I prefer `vw` and `vh` (see https://www.w3schools.com/cssref/css_units.asp) but this should be fine since CSS is not final.